src/test/java/guitests/PersonEditGuiTest.java

```diff
@@ -76,16 +73,20 @@ public void editPerson_usingAccelerator() {
     @Test
     public void editPerson_usingContextMenu() {
         personListPanel.rightClickOnPerson(td.alice);
-        EditPersonDialogHandle editPersonDialog = personListPanel.clickOnContextMenu(
-                PersonListPanelHandle.ContextMenuChoice.EDIT);
+        EditPersonDialogHandle editPersonDialog =
+                personListPanel.clickOnContextMenu(PersonListPanelHandle.ContextMenuChoice.EDIT);
         assertTrue(editPersonDialog.isValidEditDialog());
+
+        //Rest of the edit process tested in editPerson_usingAccelerator
     }
 
     @Test
     public void editPerson_usingEditButton() {
         personListPanel.clickOnPerson(td.alice);
         EditPersonDialogHandle editPersonDialog =  personListPanel.clickEdit();
         assertTrue(editPersonDialog.isValidEditDialog());
```

Comment:
The test is weak. It should check if it shows the edit dialog for the correct person.